Gemfile.lock

```diff
@@ -588,7 +588,7 @@ GEM
       rack
     orm_adapter (0.5.0)
     parallel (1.22.1)
-    parser (3.1.3.0)
```

Comment:
Created a [separate PR](https://github.com/forem/forem/pull/18935) with the rubocop update but pulled that branch in because i these files were all erroring.